rsnapshot-program.pl

```diff
@@ -32,27 +32,20 @@
 ###         STANDARD MODULES         ###
 ########################################
 
-require 5.004;
+require 5.012000; # first version where POSIX::lchown is documented
```

Comment:
This seems to work on as early as v5.8. Not sure about 5.4, 5.6 yet.

Reply:
Not sure about ealier versions because the build doesn't work on \<= 5.6 -- https://github.com/sgpinkus/rsnapshot/actions/runs/12740841950/job/35506597363#step:7:92